python/pyspark/streaming/tests.py

```diff
@@ -616,6 +615,7 @@ def test_kafka_stream(self):
 
         self._kafkaTestUtils.createTopic(topic)
         self._kafkaTestUtils.sendMessages(topic, sendData)
+        time.sleep(5)
```

Comment:
If the `waitUntilLeaderOffset` has been added to Scala, shouldnt they be added to Python as well? Maybe that then this arbitrary sleep wont be needed.